ui/components/imageWorkspace/src/ImageWorkspace.svelte

```diff
@@ -104,6 +109,30 @@
       onSave().catch((err) => console.error(err));
     }
   }
+
+  onMount(() => {
+    //TMP: should be removed when available value written in db.json, so present in DatasetInfo[] from getDatasets (S)
+    // but right now, we compute available feats only in getDataset (no S) route, used only for stats, and here
+    // later, we should get itemFeaturesAvailableValues directly from DatasetInfo["available_feat_values"]
+    api
+      .getDataset(currentDatasetId)
+      .then((datasetWithFeats) => {
+        const inputFeaturesAvailableValues: InputFeaturesAvailableValues =
+          datasetWithFeats.available_feat_values || {};
+        let outputFeaturesAvailableValues: FeaturesAvailableValues = {};
+        //convert from InputFeaturesAvailableValues to FeaturesAvailableValues
+        for (let table in inputFeaturesAvailableValues) {
```

Comment:
Following our conversation, you could have used a reduce like so : 
```
        const outputFeaturesAvailableValues: FeaturesAvailableValues = Object.entries(
          inputFeaturesAvailableValues,
        ).reduce((tables, [tableName, tableValues]) => {
          tables[tableName] = tableValues.reduce(
            (features, feature) => {
              if (feature.values) {
                features[feature.name] = feature.values;
              }
              return features;
            },
            {} as FeaturesAvailableValues[""],
          );
          return tables;
        }, {} as FeaturesAvailableValues);
```

I personnaly tend to prefer reduce but a for in loop works also well!